Cyclic_Permutation/Cyclic_Permutation.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+CYCLIC PERMUTATIONS
+
+Cyclic permutations of a number are defined as a number that
+can be obtained by rotating the concerned number any number 
+of times between it's digits.
+*/
+
+#include<bits/stdc++.h>
+using namespace std;
+
+int getDigits(int n) {
+    int ans = 0;
+    while (n) {
+        ans ++;
+        n /= 10;
+    }
+    return ans;
+}
+
+void cyclic(int n) {
+    int length = getDigits(n);
+    int res = n;
+    while (1) {
+        cout<<res<<endl;
+        int r = res % 10;
+        int d = res / 10;
+        res = r * pow(10, length - 1) + d;
+        if (res == n) {
+            break;
+        }
+    }
+}
+
+int main() {
+    cout<<"Enter N:";
+    int n;
+    cin>>n;
```

Comment:
Space around `<<`;

Reply:
Hi. @marwahmanbir had asked to remove spaces around << and >> for cin and cout statements in PRs.

Reply:
What should I do @jainaman224 ?